File: java-programming/turnOnCandle.java
class Main{
    public static boolean turnOnCandle(boolean status){
	if(!status){
	    return true;
	}
	else{
	    return false;
	}
    }
    public static void main(String[] args){
	boolean candleIsOn = false;
	if(turnOnCandle(candleIsOn)){
	    System.out.println("Candle is On!.");
	}else{
	    System.out.println("fail");
	}
    }
}

File: c-programming/turnOnCandle.c
#include <stdio.h>
#include <stdbool.h>

bool turnOnCandle(bool status){
  if (status == false){
    return true;
  }
  else{
    return false;
  }
}
int main(){
  bool candleIsOn = false;
  
  if(turnOnCandle(candleIsOn)){
    printf("The candle is on\n");
    return 0;
  }else{
    return 1;
  }
   
}

